src/sentry/middleware/user.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import
+
+import inspect
+from datetime import timedelta
+from django.utils import timezone
+
+
+class UserActiveMiddleware(object):
+    disallowed_paths = (
+        'sentry.web.frontend.generic.static_media',
+        'sentry.web.frontend.user_avatar',
+    )
+
+    def process_view(self, request, view_func, view_args, view_kwargs):
+        view = view_func
+        if not inspect.isfunction(view_func):
+            view = view.__class__
+
+        try:
+            path = '%s.%s' % (view.__module__, view.__name__)
+        except AttributeError:
+            return
+
+        if path.startswith(self.disallowed_paths):
+            return
+
+        if not request.user.is_authenticated():
```

Comment:
you could also do if hasattr(request, 'user') as I assume thats the issue you hit w/ the static paths